nix/default.nix

```diff
@@ -45,7 +57,7 @@ let
 
   self = with pkgs; stdenv.mkDerivation rec {
     name = "crun";
-    src = builtins.filterSource (p: t: lib.cleanSourceFilter p t && baseNameOf p != ".cache") ./..;
```

Comment:
why do we need to drop this part?  Without it, we copy the entire cache in the build

Reply:
> Without it, we copy the entire cache in the build

Without rollback my local build error as below:
```
builder for '/nix/store/w860c8d33a1813hb11w1njzwcgmv82px-crun.drv' failed with exit code 2; last 10 log lines:
    CC       src/crun-crun.o
  src/crun.c:20:10: fatal error: git-version.h: No such file or directory
     20 | #include <git-version.h>
        |          ^~~~~~~~~~~~~~~
  compilation terminated.
  make[2]: *** [Makefile:1532: src/crun-crun.o] Error 1
  make[2]: Leaving directory '/tmp/nix-build-crun.drv-0/crun'
  make[1]: *** [Makefile:1839: all-recursive] Error 1
  make[1]: Leaving directory '/tmp/nix-build-crun.drv-0/crun'
  make: *** [Makefile:815: all] Error 2
[6 built (1 failed), 3 copied (15.3 MiB), 2.5 MiB DL]
error: build of '/nix/store/w860c8d33a1813hb11w1njzwcgmv82px-crun.drv' failed
```

Reply:
@giuseppe I am now trying to cache `/nix` directly, rather than placing the cache under `./.cache`